projects/packages/forms/src/blocks/contact-form/edit.js

```diff
@@ -72,332 +72,299 @@ const PRIORITIZED_INSERTER_BLOCKS = [ ...map( validFields, block => `jetpack/${
-								href={ CUSTOMIZING_FORMS_URL }
-								target="_blank"
-							>
-								{ __( 'Learn more about customizing forms', 'jetpack-forms' ) }
-							</Button>
-							<Button
-								variant="link"
-								className="form-placeholder__external-link"
-								href={ RESPONSES_PATH }
-								target="_blank"
-							>
-								{ __( 'View and export your form responses here', 'jetpack-forms' ) }
-							</Button>
-						</div>
-					</div>
-					{ isPatternsModalOpen && (
-						<Modal
-							className="form-placeholder__patterns-modal"
-							title={ __( 'Choose a pattern', 'jetpack-forms' ) }
-							closeLabel={ __( 'Cancel', 'jetpack-forms' ) }
-							onRequestClose={ () => setIsPatternsModalOpen( false ) }
+						<Button
+							variant="link"
+							className="form-placeholder__external-link"
+							href={ RESPONSES_PATH }
+							target="_blank"
 						>
-							<BlockPatternSetup
-								initialViewMode="grid"
-								filterPatternsFn={ pattern => {
-									return pattern.content.indexOf( 'jetpack/contact-form' ) !== -1;
-								} }
-								clientId={ clientId }
-							/>
-						</Modal>
-					) }
+							{ __( 'View and export your form responses here', 'jetpack-forms' ) }
+						</Button>
+					</div>
 				</div>
-			);
-		};
-
-		let elt;
-
-		if ( ! isModuleActive ) {
-			if ( isLoadingModules ) {
-				elt = <ContactFormSkeletonLoader />;
-			} else {
-				elt = (
-					<ContactFormPlaceholder
-						changeStatus={ changeStatus }
-						isModuleActive={ isModuleActive }
-						isLoading={ isChangingStatus }
-					/>
-				);
-			}
-		} else if ( ! hasInnerBlocks && registerBlockVariation ) {
-			elt = renderVariationPicker();
-		} else {
-			elt = (
-				<>
-					<InspectorControls>
-						{ ! attributes.formTitle && (
-							<PanelBody>
-								<Notice status="warning" isDismissible={ false }>
-									{ __(
-										'Add a heading inside the form or before it to help everybody identify it.',
-										'jetpack-forms'
-									) }
-								</Notice>{ ' ' }
-							</PanelBody>
-						) }
-						<PanelBody title={ __( 'Manage Responses', 'jetpack-forms' ) }>
-							<JetpackManageResponsesSettings setAttributes={ setAttributes } />
-						</PanelBody>
-						<PanelBody title={ __( 'Submission Settings', 'jetpack-forms' ) } initialOpen={ false }>
-							{ renderSubmissionSettings() }
+				{ isPatternsModalOpen && (
+					<Modal
+						className="form-placeholder__patterns-modal"
+						title={ __( 'Choose a pattern', 'jetpack-forms' ) }
+						closeLabel={ __( 'Cancel', 'jetpack-forms' ) }
+						onRequestClose={ () => setIsPatternsModalOpen( false ) }
+					>
+						<BlockPatternSetup
+							initialViewMode="grid"
+							filterPatternsFn={ pattern => {
+								return pattern.content.indexOf( 'jetpack/contact-form' ) !== -1;
+							} }
+							clientId={ clientId }
+						/>
+					</Modal>
+				) }
+			</div>
+		);
+	} else {
+		const style = window.jetpackForms.generateStyleVariables( innerRef.current );
```

Comment:
This is the `withStyleVariables` HOC moved in line. It's nice that it can be moved into this branch of the `if` statement.